js/storage.js

```diff
@@ -1117,6 +1117,18 @@ Storage.exportAllTeams = function () {
 	}
 	return buf;
 };
+Storage.exportFolder = function (folder) {
+	var buf = '';
+	for (var i = 0, len = Storage.teams.length; i < len; i++) {
+		var team = Storage.teams[i];
+		if ((folder.indexOf("/") && team.folder === folder.slice(0, -1)) || team.format === folder) {
```

Comment:
`folder.indexOf("/")` and `folder.slice(0, -1)` are weird things to do in an inner loop.

Not to mention `folder.indexOf("/")` should probably be `folder.slice(-1) === '/'`